etc/profile-a-l/dino-im.profile

```diff
@@ -0,0 +1,14 @@
+# Firejail profile for dino-im
+# Description: Modern XMPP Chat Client using GTK+/Vala, Ubuntu specific bin name
+# This file is overwritten after every install/update
+##quiet
```

Comment:
I think we can drop this line.

Reply:
You mean the ``##quiet``? I was confused about it, because it is in the template, but not in other profiles I looked at.